AnkiDroid/src/main/java/com/ichi2/utils/JSONObject.kt

```diff
@@ -307,22 +306,21 @@ open class JSONObject : org.json.JSONObject, Iterable<String?> {
 
     /** deep clone this into clone.
      *
-     * Given a subtype [T] of JSONObject, and a JSONObject [j], we could do
+     * Given a subtype [T] of JSONObject, and a JSONObject [clone], we could do
      * ```
      * T t = new T();
      * j.deepClonedInto(t);
```

Comment:
```suggestion
     * clone.deepClonedInto(t);
```

just a comment-code fix. Fine with merge after this fix

Reply:
I just fixed this locally so it won't take conflicts again. going to push force-with-lease and merge it after it passes CI tasks locally on a rebase from main